Swiftfin/Views/HomeView/HomeView.swift

```diff
@@ -87,11 +90,19 @@ struct HomeView: View {
                 ProgressView()
             }
 
-            SettingsBarButton(
-                server: viewModel.userSession.server,
-                user: viewModel.userSession.user
-            ) {
-                mainRouter.route(to: \.settings)
+            HStack(spacing: 0) {
+                if activeSessionIndicator {
+                    ActiveSessionIndicator {
+                        print("This will take you to the Admin Dashboard once it's on its own coordinator...")
```

Comment:
Is this print left behind intentionally?

Reply:
It was not haha. Sorry about that, meant to flip that to a TODO for later. Consider it gone.